tests/testTranslationRecovery.cpp

```diff
@@ -49,15 +49,17 @@ TEST(TranslationRecovery, BAL) {
       poses, {{0, 1}, {0, 2}, {1, 2}});
 
   // Check
-  const Pose3 wTa = poses.at<Pose3>(0), wTb = poses.at<Pose3>(1),
-              wTc = poses.at<Pose3>(2);
-  const Point3 Ta = wTa.translation(), Tb = wTb.translation(),
-               Tc = wTc.translation();
-  const Rot3 aRw = wTa.rotation().inverse();
-  const Unit3 w_aZb = relativeTranslations.at({0, 1});
-  EXPECT(assert_equal(Unit3(Tb - Ta), w_aZb));
-  const Unit3 w_aZc = relativeTranslations.at({0, 2});
-  EXPECT(assert_equal(Unit3(Tc - Ta), w_aZc));
+  Unit3 w_aZb_stored; // measurement between 0 and 1 stored for next unit test
```

Comment:
Because we need to use it in line 74. 
The old code stores both w_aZb and w_aZc. Ive renamed it to w_aZb_stored because I use w_aZb in the for loop for something else. I agree its not a great name. 
Storing from a map is easy because we can always look up using "at". But since it is a vector, we need to iterate over the edges to look up. I'm doing the iteration in lines 53-62 for another unit test, and at the same time storing this values.